webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/DataSourceService.scala

```diff
@@ -226,7 +226,10 @@ class DataSourceService @Inject()(
       val uri = new URI(pathStr)
       if (DataVaultService.isRemoteScheme(uri.getScheme)) true
       else {
-        val path = Path.of(new URI(pathStr).getPath).normalize().toAbsolutePath
+        val path = organizationDir
+          .resolve(dataSource.id.directoryName)
+          .resolve(Path.of(new URI(pathStr).getPath).normalize())
+          .toAbsolutePath
```

Comment:
What about the directory of the layer? Like 
```
val path = organizationDir
          .resolve(dataSource.id.directoryName)
          .resolve(dataLayerName)
          .resolve(Path.of(new URI(pathStr).getPath).normalize())
          .toAbsolutePath
```

Reply:
Just checked: Absolute and relative Paths. Both worked, but non-existent paths were accepted